server/action_creators.js

```diff
@@ -80,6 +80,17 @@ export function addEntriesToOrder(orderId, newEntries) {
   };
 }
 
+export function setReadyForBill(orderId) {
+  return (dispatch, getState) => {
+    return Order.findOneAndUpdate({ id: orderId }, { status: constants.READY_FOR_BILL })
+    .then(() => {
+      return Order.getOrders();
```

Comment:
I think it will generally be okay if we incrementally update all affected parts of the state. But this way is definitely a lot cleaner and avoids that other `findIndex` BS in a couple of the other actions. Just thought we might be concerned about doubling the amount of database requests.

If we decide to do it this way, where we just fetch all, we don't even need server store / state...